### PR TITLE
Privacy Considerations

### PR DESCRIPTION
**Pull request by Wind4Greg**
This PR furnishes a **privacy considerations** section for the VC-DI-BBS draft. It focuses on two key issues: (i) data leakage and (ii) unlinkability. It defines the issues, shows examples of how they could arise, discusses how they are mitigated in the specification, and, in the case of unlinkability, explains the roles of the issuer and holders in maintaining unlinkability.

***

### :boom: Error: 500 Internal Server Error :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Dec 15, 2023, 3:21 AM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [Spec Generator](https://www.w3.org/2015/labs/) - Spec Generator is the web service used to build specs that rely on ReSpec.

:link: [Related URL](https://labs.w3.org/spec-generator/?type=respec&url=https%3A%2F%2Fraw.githubusercontent.com%2FWind4Greg%2Fvc-di-bbs%2F9b3b242197354f9ee5a35079e8190eee10297244%2Findex.html%3FisPreview%3Dtrue)

```

😭  Sorry, there was an error generating the HTML. Please report this issue!
Specification: http://labs.w3.org/spec-generator/uploads/7Nqe8k/index.html?isPreview=true%3FisPreview%3Dtrue&publishDate=2023-12-15
ReSpec version: 34.2.2
File a bug: https://github.com/w3c/respec/
Error: undefined

```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20w3c/vc-di-bbs%23101.)._
</details>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1300,15 +1320,440 @@ <h4>Verify Derived Proof (bbs-2023)</h4>
...
+        <section>
+          <h4>Linkage via Holder Selective Reveal</h4>
+          <p>
+As discussed in [[Powar2023]] there are many documented cases of
+re-identification of individuals from linkage attacks. Hence the <em>holder</em>
+is urged  to reveal as little information as possible to help keep the anonymity
+set large. In addition, it has been shown a number of times that innocuous
+seeming information can be highly unique and thus leading to re-identification
+or tracking.  See [[NISTIR8053]] for a walk through of a particularly famous
+case of  a former governor of Massachusetts and [[Powar2023]] for further
```
Comment (dlongley) [inline index.html:1730]:
```suggestion
case of a former governor of Massachusetts and [[Powar2023]] for further
```

File: index.html [hunk 2]
```diff
@@ -1300,15 +1320,440 @@ <h4>Verify Derived Proof (bbs-2023)</h4>
...
+Although the ZKPKS, known as a <em>BBS proof</em> in this document, has
+guaranteed unlinkability properties. BBS when used with selective disclosure has
+two artifacts that can contribute to linkability. These are the total number of
+messages originally signed, and the index values for the revealed statements.
+See the privacy considerations in [[CFRG-BBS-SIGNATURE]] for a discussion and
+mitigation techniques.
+          </p>
+          <p>
+As mentioned in the section on <q>Issuer's Public Keys</q> of
+[[CFRG-BBS-SIGNATURE]] there is the potential threat that an <em>issuer</em> may
```
Comment (dlongley) [inline index.html:1670]:
```suggestion
[[CFRG-BBS-SIGNATURE]] there is the potential threat that an <em>issuer</em> might
```

File: index.html [hunk 3]
```diff
@@ -1300,15 +1320,440 @@ <h4>Verify Derived Proof (bbs-2023)</h4>
...
+the structure of data or from the lower level cryptographic primitives.
+        </p>
+        <p>
+For example the BBS signature scheme is an extremely space efficient scheme for
+producing a signature on multiple <em>messages</em>, i.e.,  the cryptographic
+signature sent to the <em>holder</em> is a constant size regardless of the
+number of <em>messages</em>. The <em>holder</em> then can selectively disclose
+any of these <em>messages</em> to a <em>verifier</em>, however as part of the
+encryption scheme, the total number of messages signed by the <em>issuer</em>
+must be revealed to the <em>verifier</em>. If such information leakage needs to
```
Comment (dlongley) [inline index.html:1341]:
```suggestion
has to be revealed to the <em>verifier</em>. If such information leakage needs to
```

**Review by dlongley — APPROVED** [review]
+1 to get this section in and allow for additional iteration as needed. Thanks!

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
@Wind4Greg there are now merge conflicts because of the security considerations merge. Could you please fix the merge conflicts so we can merge this PR?

**Comment by msporny** [comment]
@Wind4Greg There are merge conflicts here and I don't know how to resolve them. Something terrible has happened and I don't want to lose the history in this PR. Could you please try to rebase on main and create a clean PR? The merge conflicts arose when the Security Considerations section was merged to main.

**Pull request closed without merging**